packages/pluggableWidgets/maps-web/package.json

```diff
@@ -34,7 +34,8 @@
     "@types/googlemaps": "^3.39.2",
     "@types/faker": "^4.1.0",
     "@types/react-leaflet": "^2.5.1",
-    "faker": "^4.1.0"
+    "faker": "^4.1.0",
+    "cross-env": "^7.0.2"
```

Comment:
Alphabetical :P

Reply:
Meh! This should be staged-automated :P

Reply:
https://stackoverflow.com/questions/34438465/is-there-a-way-to-alphabetize-package-json-without-installing-a-package